src/profile/EditProfile.js

```diff
@@ -409,49 +393,10 @@ export default class EditProfile extends Component {
 
         <br />
 
-        {/* Here we are updating our default language */}
-
-        <FormGroup controlId="defaultLanguage" bsSize="large">
-          <ControlLabel>Default Language</ControlLabel>
-          <div className="flex">
-            <FormControl
-              componentClass="select"
-              onChange={this.handleChange}
-              value={this.state.defaultLanguage}
-            >
-              <option value="en">Choose Here</option>
-              <option value="en">English</option>
-              <option value="lg">Luganda</option>
-              <option value="ac">Acholi</option>
-              <option value="es">Spanish</option>
-              <option value="ptbr">Portuguese</option>
-              <option value="hi">Hindi</option>
-            </FormControl>
-            <LoaderButton
-              align="center"
-              block
-              type="submit"
-              style={{ width: 90 }}
-              // disabled={!this.validateForm()}
-              onClick={this.updateLanguage}
-              isLoading={this.state.isLoading}
-              text="Save"
-              loadingText="Updating..."
-            />
-          </div>
-        </FormGroup>
-
-        <Button onClick={() => this.props.history.push("/settings/password")}>
-          Change Password
-        </Button>
-
-        <Tour
-          steps={steps}
-          isOpen={this.state.isTourOpen}
-          onRequestClose={this.closeTour}
-        // showCloseButton={true}
-        // rewindOnClose={false}
-        />
+           <Languages  
+             id = {this.state.id}
+             user = {this.user}
```

Comment:
I don't think this is correct - shouldn't it be `this.state.user`? I don't remember there being a `this.user` object in the constructor of the `EditProfile` component.

Reply:
@mikhael28  ok, those issues was resolved. you were right about the this.state.user prop being passed don't know how I didn't see that. The props.value on line 98 wasn't needed because the function updated the label to its current value. I did remove the consoles, but I will send you the GIF and another revised PR tomorrow. Thanks